src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -219,6 +221,11 @@ internal sealed override ImmutableArray<string> NotNullWhenFalseMembers
             var hasBody = syntax.Body != null;
             var hasExpressionBody = syntax.ExpressionBody != null;
             _isExpressionBodied = !hasBody && hasExpressionBody;
+            _isInitOnly = syntax.Keyword.IsKind(SyntaxKind.InitKeyword);
```

Comment:
>_isInitOnly [](start = 12, length = 11)

I think we shouldn't set this to ```true``` for a static accessor. We also can report ERR_BadInitAccessor here.

Reply:
Adding a PROTOTYPE marker in relevant test to discuss further

---
In reply to: [415126177](https://github.com/dotnet/roslyn/pull/43275#discussion_r415126177) [](ancestors = 415126177)

Reply:
Please add a test that asserts that IsInotIOnly doesn't return true for a static accessor and that no modreqs add for it. 

---
In reply to: [415212881](https://github.com/dotnet/roslyn/pull/43275#discussion_r415212881) [](ancestors = 415212881,415126177)